packages/components/stories/DateTimePicker.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import {
+	DateTimeWrapper,
+	MonthYearWrapper,
+	DateTimePicker,
+} from '../src/DateTimePickers';
+
+
+storiesOf('DateTimePicker', module)
+	.add('DateTimePicker structure', () => (
+		<div>
+			<h1>DateTimePickers structure</h1>
+			<DateTimePicker />
+		</div>
+	))
+	.add('Date/Time picker structure', () => (
+		<div>
+			<h1>Date/Time structure</h1>
+			<DateTimeWrapper />
+		</div>
+	))
+	.add('Month/Year picker structure', () => (
+		<div>
```

Comment:
Can you set a sze to the container that is the min width we should support  ? (320px)